.github/workflows/build_workflow.yml

```diff
@@ -52,6 +52,7 @@ jobs:
     strategy:
       matrix:
         python-version: ["3.9", "3.10", "3.11", "3.12", "3.13"]
+        fail-fast: false
```

Comment:
@altheaden, it seems this wasn't indented properly.  It should be in strategy but not in matrix, see:
https://github.com/E3SM-Project/mache/blob/main/.github/workflows/build_workflow.yml#L63

Could you make a PR to fix this when you have a chance?

Reply:
Whoops, sure thing! Sorry about that.